packages/publikator/migrations/sqls/20210201080354-base-down.sql

```diff
@@ -0,0 +1,8 @@
+DROP TRIGGER IF EXISTS trigger_publikator_commits_notify_change
+  ON publikator.commits;
+DROP TRIGGER IF EXISTS trigger_publikator_repos_notify_change
+  ON publikator.repos;
+DROP TABLE publikator.milestones;
+DROP TABLE publikator.commits;
+DROP TABLE publikator.repos;
```

Comment:
In up script we have `ALTER TABLE "public"."notifyTableChangeQueue" ALTER COLUMN "id" TYPE text;`, shouldn't it be changed back in down script too?

Reply:
Indeed!